lib/hpcbase.pm

```diff
@@ -36,44 +36,44 @@ sub switch_user {
 ## Prepare master node names, so those names could be reused, for instance
 ## in config preparation, munge key distribution, etc.
 ## The naming follows general pattern of master-slave
-sub ctl_node_names {
-    my $ctl_nodes = get_required_var("MASTER_NODES");
-    my @ctl_node_names;
+sub master_node_names {
```

Comment:
Well master/slave naming is was originally in our openQA tests, but proper upstream naming is controller and compute node. Please check https://slurm.schedmd.com/quickstart.html.

Reply:
Sure, I'm aware of the naming convention for slurm.
In the commit message I wrote: "HPC: Generalize cluster naming
HPC cluster naming is generalized so that the nodes are named always
master or slave. This approach should allow reusing the code easier".

So that's the reasoning I diverge from the initial idea of using ctl/compute for slurm.

Wdyt? 

Practical use -> remove that for loop to manage munge key distribution and have it as a generic function in the hpcbase.pm

Reply:
I would like follow slurm upstream naming, but i'm not against existing combination. And if you think,  that it is better, we can keep it.